core/templates/pages/contributor-dashboard-page/services/translate-text.service.ts

```diff
@@ -76,7 +76,7 @@ export class TranslateTextService {
 
   constructor(
     private translateTextBackedApiService:
-      TranslateTextBackendApiService
+      translateTextBackendApiService
```

Comment:
This was already correct. The typo is with `translateTextBackedApiService` on lines 78, 162, 228.